app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Dialog.kt

```diff
@@ -66,9 +142,11 @@ fun HedvigDialog(
     onDismissRequest = onDismissRequest,
     properties = DialogDefaults.defaultProperties,
   ) {
-    (LocalView.current.parent as DialogWindowProvider).window.setDimAmount(0.2f)
-    // a workaround to stop the overlay from dimming background too much, otherwise in the dark theme the overlay color
+    // (LocalView.current.parent as DialogWindowProvider).window.setDimAmount(0.2f)
+    // a workaround to stop the overlay from dimming background too much,
+    // otherwise in the dark theme the overlay color
     // becomes the same as the background color of the dialog itself.
+    // todo: that workaround stopped working btw
```

Comment:
well, now we are not doing anything, and it kinda works (the color is not completely the same in the dark mode), but it's still very-very close and hard to distinguish. But again, this is dark mode - maybe it's supposed to be so, idk.

Reply:
Alright, yeah let's not do anything about it then, and remove the workaround code since it doesn't work for now, and leave it be unless someone complains about it in the first place